test/gateway/eip20_cogateway/redeem.js

```diff
@@ -227,17 +249,17 @@ contract('EIP20CoGateway.redeem() ', function (accounts) {
 
   it('should fail when the redeemer\'s BT balance is less than the redeem amount', async function () {
 
-    let amount = new BN(10000);
+    let redeemAmount = amount.muln(2);
```

Comment:
Why not `= balance.addn(1)` where `balance` is the redeemer's BT balance?

Reply:
done 👍